shared/chat/conversation/input-area/index.stories.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+import * as React from 'react'
+import {Set} from 'immutable'
+import {Input as TextInput} from '../../../common-adapters'
+import {Box2} from '../../../common-adapters/box'
+import {platformStyles} from '../../../styles'
+import * as PropProviders from '../../../stories/prop-providers'
+import {action, storiesOf} from '../../../stories/storybook'
+import mentionHoc, {type PropsFromContainer} from './mention-handler-hoc'
+import {default as _Input} from './normal'
+import {isMobile} from '../../../constants/platform'
+import {stringToConversationIDKey} from '../../../constants/types/chat2'
+
+// For some reason, flow can't infer the type of mentionHoc here.
+const Input: React.ComponentType<PropsFromContainer> = mentionHoc(_Input)
+
+const provider = PropProviders.compose(PropProviders.Usernames(['max', 'cnojima', 'cdixon'], 'ayoubd'), {
+  ChannelMentionHud: ownProps => {
+    const channels = ['foo', 'bar']
+    return {
+      ...ownProps,
+      channels,
+    }
+  },
+  UserMentionHud: ownProps => {
+    const users = [
+      {username: 'marcopolo', fullName: 'Marco Munizaga'},
+      {username: 'trex', fullName: 'Thomas Rex'},
+      {username: 'chris', fullName: 'Chris Coyne'},
+    ]
+    return {
+      ...ownProps,
+      users,
+    }
+  },
+})
+
+type Props = {
+  isEditing: boolean,
+  pendingWaiting: boolean,
+  text: string,
+  typing: Set<string>,
+}
+
+type State = {
+  text: string,
+}
+
+// On mobile, we want full width and height. On desktop, we we want to
+// manually set dimensions. Not sure why fullHeight doesn't work on
+// mobile, though.
```

Comment:
tried for a while, but couldn't get `fullHeight` to work on mobile -- had to set it manually.